man/froll.Rd

```diff
@@ -12,71 +12,67 @@
+  \item{align}{ Character, specifying the "alignment" of the rolling window, defaulting to \code{"right"}. \code{"right"} covers preceding rows (the window \emph{ends} on the current value); \code{"left"} covers following rows (the window \emph{starts} on the current value); \code{"center"} is halfway in between (the window is \emph{centered} on the current value, biased towards \code{"left"} when \code{n} is even). }
+  \item{na.rm}{ Logical, default \code{FALSE}. Should missing values be removed when
+    calculating window? For details on handling other non-finite values, see Details. }
+  \item{hasNA}{ Logical. If it is known that \code{x} contains \code{NA}
+    then setting this to \code{TRUE} will speed up calculation. Defaults to \code{NA}. }
+  \item{adaptive}{ Logical, default \code{FALSE}. Should the rolling function be calculated adaptively? See Details below. }
+  \item{FUN}{ The function to be applied to the rolling window; see Details for restrictions. }
+  \item{\dots}{ Extra arguments passed to \code{FUN} in \code{frollapply}. }
 }
 \details{
-  \code{froll*} functions accepts vectors, lists, data.frames or
-  data.tables. They always return a list except when the input is a
-  \code{vector} and \code{length(n)==1} in which case a \code{vector}
-  is returned, for convenience. Thus rolling functions can be used
-  conveniently within data.table syntax.
+  \code{froll*} functions accept vectors, lists, \code{data.frames} or
+  \code{data.tables}. They always return a list except when the input is a
+  \code{vector} and \code{length(n)==1}, in which case a \code{vector}
+  is returned, for convenience. Thus, rolling functions can be used
+  conveniently within \code{data.table} syntax.
 
   Argument \code{n} allows multiple values to apply rolling functions on
-  multiple window sizes. If \code{adaptive=TRUE}, then it expects a list.
+  multiple window sizes. If \code{adaptive=TRUE}, then \code{n} must be a list.
   Each list element must be integer vector of window sizes corresponding
-  to every single observation in each column.
+  to every single observation in each column; see Examples.
 
-  When \code{algo="fast"} then \emph{on-line} algorithm is used, also
-  any \code{NaN, +Inf, -Inf} is treated as \code{NA}.
-  Setting \code{algo="exact"} will make rolling functions to use
-  compute-intensive algorithm that suffers less from floating point
-  rounding error. It also handles \code{NaN, +Inf, -Inf} consistently to
+  When \code{algo="fast"} an \emph{"on-line"} algorithm is used, and
+  all of \code{NaN, +Inf, -Inf} are treated as \code{NA}.
+  Setting \code{algo="exact"} will make rolling functions to use a more
+  computationally-intensive algorithm that suffers less from floating point
+  rounding error (the same consideration applies to \code{\link[base]{mean}}).
+  \code{algo="exact"} also handles \code{NaN, +Inf, -Inf} consistently to
   base R. In case of some functions (like \emph{mean}), it will additionally
```

Comment:
Jan this second sentence looks like the same as the one before, is it redundant or am I missing something?

Reply:
What do you mean by the same applies to base mean? Afair base mean do rounding correction always.
If we don't want to repeat anything then better remove the one before.

Reply:
Just to convey that `algo='exact'` should be matching `base::mean` which does error correction right?